ApplicationServerDeployment/lambda/alexa/response.py

```diff
@@ -0,0 +1,185 @@
+import random
+import uuid
+import datetime
+import logging
+logger = logging.getLogger(__name__)
+class AlexaResponse:

```

Comment:
add block comment to explain the purpose of the class

Reply:
done